### ISSUE TITLE
does Phase 2 have correct answer?

### ISSUE THREAD

**Issue by Haotian-Shi-cyber** [magna25/Attack-Lab#10]
ive tried several times but still failed and it seems like old rsp value should be on the bottom of answer

**Comment by Kenan-Kamel** [comment]
You are right, the value of the rsp register should be push just before the value of the touch2 function, and in some cases, you might need to double push it if you have a segfault error.
